tests/test_io.py

```diff
@@ -438,14 +441,17 @@ def test_load_scan_topostats(load_scan_topostats: LoadScans) -> None:
     image, px_to_nm_scaling = load_scan_topostats.load_topostats()
     grain_masks = load_scan_topostats.grain_masks
     above_grain_mask = grain_masks["above"]
+    grain_trace_data = load_scan_topostats.grain_trace_data
```

Comment:
These lines updated from re-creating the demo `.topostats` file `file.topostats` since we are changing a part of how they are generated, to keep it consistent.